src/pages/part8.jsx

```diff
@@ -61,11 +61,14 @@ const Part8 = () => {
           Sign Up For Out Mailing List
         </Typography>
       {/* TODO: (3) Change the p tags to Typography with variant="body1" */}
+        <Typography variant="body1" alight="center">
```

Comment:
The Typography component shouldn't have paragraphs inside it, it *is* the paragraph.
```jsx
<div>
    <Typography variant="body1">Text here</Typography>
    <Typography variant="body1">Text here</Typography>
</div>
```